manifests/root_user.pp

```diff
@@ -10,10 +10,14 @@
 # @param manage_group
 #  Ensure the root group has appropriate UIDs, etc
 #
+# @param password
+#  Set the root user's password using Puppet
+#
 class simp::root_user (
-  Boolean $manage_perms = true,
-  Boolean $manage_user  = true,
-  Boolean $manage_group = true
+  Boolean           $manage_perms = true,
+  Boolean           $manage_user  = true,
+  Boolean           $manage_group = true,
+  Optional[String]  $password     = undef,
```

Comment:
Should we auto-hash this if they provide a plain text password? If not, should we at least validate that this is in a correct format for the password file?

Ref: https://puppet.com/docs/puppet/5.3/types/user.html#user-attribute-password

Reply:
Auto hash addded.